prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/specs/AddCourtAppointmentSpecification.groovy

```diff
@@ -59,9 +58,9 @@ class AddCourtAppointmentSpecification extends BrowserReportingSpec {
         at SelectCourtAppointmentRoomsPage
 
         and: "I fill out the form"
-        selectRoomsForm.preAppointmentLocation = 1
-        selectRoomsForm.mainAppointmentLocation = 2
-        selectRoomsForm.postAppointmentLocation = 3
+        selectRoomsForm.selectPreAppointmentLocation = 1
```

Comment:
There is some Javascript that's always loaded that's used by a different page that has some pre and post elements on it, however there are change events hooked into the ids listed above. So for now I've just renamed them to avoid clashes. Longer time we either need to reduce the Javascript, reduce the scope with a named class, or load the js for specific pages.

Reply:
We do classes prefixed with `js-` for the other appointment stuff so makes sense to carry on doing that.